learningresources/models.py

```diff
@@ -19,6 +19,18 @@
 log = logging.getLogger(__name__)
 
 
+def static_asset_basepath(asset, filename):
+    """
+    Creates folder base path for given asset.
```

Comment:
Since this is used outside this file, can you add the args and return value for this?